.github/workflows/check-valid-env-dev.yml

```diff
@@ -0,0 +1,19 @@
+name: env-dev.yml
+on:
+  pull_request:
+    paths:
+      - 'env-dev.yml'
+jobs:
+  check:
+    name: Check Env Dev
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Env-dev Valid
+        uses: mamba-org/setup-micromamba@v1.7.0
```

Comment:
Any third-party action (not ACCESS-NRI and not GitHub) we should use a commit hash rather than a tag, e.g.

https://github.com/ACCESS-NRI/reproducibility/blob/main/.github/workflows/generate-initial-checksums.yml#L175

For security reasons.

Reply:
Thanks, I've added this in